packages/cli/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [4.4.2] - 2024-03-28
```

Comment:
I think its missing from the changelog but there is support for multichain deployments now. It would make this a minor release